### PR TITLE
Add `tokio` to the list of dependencies for the README's example

### PR DESCRIPTION
**Pull request by float3**
#2220

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ use serenity::model::channel::Message;
 use serenity::framework::standard::macros::{command, group};
 use serenity::framework::standard::{StandardFramework, CommandResult};
 
+use tokio;
+
```
Comment (arqunis) [inline README.md]: This isn't necessary. Since Rust 2018, crates get automatically imported to all modules in a crate.
```suggestion
```

**Review by arqunis — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -97,6 +99,7 @@ Add the following to your `Cargo.toml` file:
 ```toml
 [dependencies]
 serenity = "0.11"
+tokio = { version = "1.21.2", features = ["full"] }
```
Comment (kangalio) [inline README.md]: It's better to enable just the features you need, so `features = ["macros"]` (`#[tokio::main]` is a macro)

**Comment by kangalio** [comment]
Thanks for the quick PR!

**Review by arqunis — APPROVED** [review]

**Comment by float3** [comment]
actually trying to compile with 
`tokio = { version = "1.21.2", features = ["tokio-macros"] }`
or
`tokio = { version = "1.21.2", features = ["macros"] }`
produces 
```
error: The default runtime flavor is `multi_thread`, but the `rt-multi-thread` feature is disabled.
  --> src\main.rs:18:1
   |
18 | #[tokio::main]
   | ^^^^^^^^^^^^^^
   |
   = note: this error originates in the attribute macro `tokio::main` (in Nightly builds, run with -Z macro-backtrace for more info)
```

**Comment by float3** [comment]
it only compiles sucessfully if I compile with `tokio = { version = "1.21.2", features = ["full"] }`

**Comment by float3** [comment]
although the docs say it should work https://docs.rs/tokio/latest/tokio/#feature-flags

**Comment by arqunis** [comment]
Ah, that's because `tokio` mandates choosing between either the single-threaded or multi-threaded runtime. I'll add the `rt-multi-thread` feature to the README myself to save you the trouble of making another PR, but I'll also credit you as the author; I should've caught the mistake when I hit "Approve".

**Comment by float3** [comment]
Thanks

**Pull request merged**